cmd/kubeapps-apis/server/server.go

```diff
@@ -65,13 +66,48 @@ func CreateRequestLogger() func(ctx context.Context, req interface{}, info *grpc
 	}
 }
 
+func getLogLevelOfEndpoint(endpoint string) klogv2.Level {
+
+	// Add all endpoint function names which you want to suppress in interceptor logging
+	supressLoggingOfEndpoints := []string{"GetConfiguredPlugins"}
+	var level klogv2.Level
+
+	// level=3 is default logging level
+	level = 3
+	for i := 0; i < len(supressLoggingOfEndpoints); i++ {
```

Comment:
It's useful having `suppressLoggingOfEndpoints` as a string slice if it's something configurable or we have more than one endpoint being hit for liveness/readiness... but otherwise, why not just do a single string comparison here?

Reply:
Today, we have only one endpoint to suppress. I guess as we test the code more, we might have a few more endpoints.

I kept it as a string slice so that In the future, we add more names to suppress. (This will be a one-time change and I don't think we need any configurable option for now.

Reply:
Yep, fine either way.